api/src/configuration.ts

```diff
@@ -12,16 +12,25 @@ export type SyncConfig = {
     tolerance: number;
 };
 
-export default () => ({
-    auth: {
-        jwtSecret: process.env.JWT_SECRET,
-    } as AuthConfig,
-    database: {
-        connectionString: process.env.DB_CONNECTION_STRING,
-        database: process.env.DB_DATABASE,
-        maxSockets: parseInt(process.env.DB_MAX_SOCKETS, 10) || 512,
-    } as DatabaseConfig,
-    sync: {
-        tolerance: parseInt(process.env.PORT, 10) || 1000,
-    } as SyncConfig,
-});
+export type Configuration = {
+    permissionMap: any;
```

Comment:
Would be nice to have some typing here, not strictly necessary though.

Reply:
We are typing the validated permission map. This property is holding the raw string value stored in the .env file